website/pages/docs/references/eslint.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: ESLint Plugin
+description: ESLint plugin for Panda CSS 
+---
+
+# ESLint Plugin
+
+ESLint plugin for Panda CSS: `@pandacss/eslint-plugin`.
+
+## Installation
+
+<Tabs items={['pnpm', 'npm', 'yarn', 'bun']} variant="code">
+  <Tab>
+    ```bash
+    pnpm add -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    npm install -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    yarn add -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    bun add -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+</Tabs>
+
```

Comment:
[Suggestion] IME it's good to also mention how to include the plugin in the ESLint config. Lots of folks are so low confidence with ESLint they'll be confused why the package manager installations on their own aren't working.

```suggestion

Something something then in your ESLint config:

\`\`\`json
"plugins": ["@pandacss/eslint-plugin"]
\`\`\`

```

If folks do see just the code block below, there's a good chance they'll skim, miss the actual explanation, and think `settings` are necessary.